common/language.c

```diff
@@ -50,6 +50,7 @@ const char* const g_strings[StrId_Max][17] =
     {
         STR_EN("● Applet Mode ●"),
         STR_ES("● Modo Applet ●"),
+        STR_FR("● Mode applet ●"),
```

Comment:
I wonder whether or not `Applet` should be capitalized or not, since it's a title

Reply:
You have the choise